src/app/routes/login.tsx

```
@@ -0,0 +1,33 @@
+import { createFileRoute, Link } from '@tanstack/react-router';
```

ESLint

[error] 'Link' is defined but never used.

(@typescript-eslint/no-unused-vars)